### PR TITLE
Exclude pyside 2 test on comprehensive workflow

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Same as #6711 but on comprehensive workflow

### CODE REVIEW

File: .github/workflows/test_comprehensive.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ on:
       - "v*x"
     tags:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:
+    paths:
+      - '.github/workflows/test_comprehensive.yml'
```
Comment (psobolewskiPhD) [inline .github/workflows/test_comprehensive.yml:14]: What's this change for?

Comment (psobolewskiPhD) [reply .github/workflows/test_comprehensive.yml:14]: Oh I get it you're making this proc on PR.

**Review by psobolewskiPhD — APPROVED** [review]

File: .github/workflows/test_comprehensive.yml (hunk 1 above)
Comment (psobolewskiPhD) [reply .github/workflows/test_comprehensive.yml:14]: I assume you will delete this after tests go green.

Comment (Czaki) [reply .github/workflows/test_comprehensive.yml:14]: Why? I think that we should ths workflow every time when someone edits it.

**Pull request merged**